config/packages/doctrine.yaml

```diff
@@ -16,6 +16,7 @@ doctrine:
         auto_generate_proxy_classes: true
         naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
         auto_mapping: true
+        enable_lazy_ghost_objects: true
```

Comment:
https://github.com/doctrine/DoctrineBundle/issues/1651 makes a comment that implies symfony 6.2+ is needed before turning this on. idk, but wanted to bring it to your attention.